op-chain-ops/genesis/config.go

```diff
@@ -819,12 +819,10 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 		"l2Usdc":             predeploys.FiatTokenV2_2Addr,
 		"l2UsdcMasterMinter": predeploys.MasterMinterAddr,
 	}
-	storage["SignatureChecker"] = state.StorageValues{}
 	storage["MasterMinter"] = state.StorageValues{
 		"_owner": config.MasterMinterOwner,
 		"controllers": map[any]any{
-			"_controller": predeploys.L2UsdcBridgeAddr,
-			"_worker":     predeploys.L2UsdcBridgeAddr,
+			predeploys.L2UsdcBridgeAddr: predeploys.L2UsdcBridgeAddr,
```

Comment:
https://github.com/tokamak-network/tokamak-thanos/blob/79ee6926b86e975101ba5b0d71d774ab224b186c/packages/tokamak/contracts-bedrock/src/tokamak-contracts/USDC/L2/tokamak-USDC/minting/Controller.sol#L70

Great. Thank you very much, Brave,,,🙇🏼‍♂️😍🥰